.github/workflows/block-autosquash-commits.yml

```diff
@@ -0,0 +1,15 @@
+on: pull_request
+
+name: Pull Requests
+
+jobs:
+  message-check:
+    name: Block Autosquash Commits
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Block Autosquash Commits
+        uses: xt0rted/block-autosquash-commits-action@v2.0.0
```

Comment:
Hmm, we were having a debate on GH actions over here:
 https://github.com/coreos/fedora-coreos-tracker/issues/764#issuecomment-796940250

Also related is https://julienrenaux.fr/2019/12/20/github-actions-security-risk/ and most recently of course https://about.codecov.io/security-update/

One approach is to pin actions by sha1.  Another though is to be sure that there are sufficient protection rules that e.g. only administrators can override status checks, so a compromised GH action can't push arbitrary code to the repo.

Does this really need the repo token?

Reply:
No idea, I just copied this from Flatpak - please change as needed.

Reply:
Try #419 if you don't like this.